Automattic/jetpack
# Sharing block/wp settings suggest deactivating module

Fixes https://github.com/Automattic/jetpack/issues/34727

## Proposed changes:

Adding a small notice below inviting users to disable Old sharing module (note only when block is available)

<img width="1115" alt="Screenshot 2023-12-21 at 11 06 18" src="https://github.com/Automattic/jetpack/assets/60262784/f7b859ea-b6a5-4cc1-a193-d1aa98aa3b01">

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






- Checkout changes locally, build `jetpack build plugins/jetpack` and spin up your site locally
- You should see new view only when beta blocks enabled: `define( 'JETPACK_BLOCKS_VARIATION', 'beta' );` and  for block-based theme (eg Twenty Twenty-Four) 
- Check `/wp-admin/options-general.php?page=sharing`.
- Check the new promt and verify that link is working
